tests/manage/test_pvc_disruptive.py

```diff
@@ -69,15 +71,33 @@ class TestRBDDisruption(BaseDisruption):
     @pytest.mark.parametrize(
         argnames=["operation_to_disrupt", "resource_to_delete"],
         argvalues=[
-            pytest.param(*['create_pvc', 'mgr'], marks=pytest.mark.polarion_id("OCS-568")),
-            pytest.param(*['create_pod', 'mgr'], marks=pytest.mark.polarion_id("OCS-569")),
-            pytest.param(*['run_io', 'mgr'], marks=pytest.mark.polarion_id("OCS-570")),
-            pytest.param(*['create_pvc', 'mon'], marks=pytest.mark.polarion_id("OCS-561")),
-            pytest.param(*['create_pod', 'mon'], marks=pytest.mark.polarion_id("OCS-562")),
-            pytest.param(*['run_io', 'mon'], marks=pytest.mark.polarion_id("OCS-563")),
-            pytest.param(*['create_pvc', 'osd'], marks=pytest.mark.polarion_id("OCS-565")),
-            pytest.param(*['create_pod', 'osd'], marks=pytest.mark.polarion_id("OCS-554")),
-            pytest.param(*['run_io', 'osd'], marks=pytest.mark.polarion_id("OCS-566")),
+            pytest.param(
+                *['create_pvc', 'mgr'], marks=pytest.mark.polarion_id("OCS-568")
```

Comment:
Looks like some spacing issue here? github shows blank lines..

Reply:
@vasukulkarni I don't see that